.github/workflows/actions.yml

```diff
@@ -28,8 +28,7 @@ jobs:
           java-version: ${{ matrix.java }}
       - name: Run JS tests
         run: |
-          ./mill -i __.resolvedIvyDeps
-          ./mill -i -j 0 all __.js.__.compile __.js.__.publishArtifacts __.js.__.test
+          ./mill -i -j 0 -k __.js.__.compile + __.js.__.publishArtifacts + __.js.__.test
```

Comment:
Mill 0.10.10 is mitigating most of the known coursier issues, so there is no need to run `__.resolvedIvyDeps` in non-parallel mode. 🤞‍